ocs_ci/templates/ocp-deployment/ntp_chrony.yaml

```diff
@@ -0,0 +1,18 @@
+# This example MachineConfig replaces /etc/chrony.conf
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 50-examplecorp-chrony
```

Comment:
Can we name this like `worker-chrony-configuration`

Reply:
done